modal/_functions.py

```diff
@@ -1126,12 +1126,8 @@ async def _load(
                 )
                 try:
                     response: api_pb2.FunctionCreateResponse = await load_context.client.stub.FunctionCreate(request)
-                except GRPCError as exc:
-                    if exc.status == Status.INVALID_ARGUMENT:
-                        raise InvalidError(exc.message)
-                    if exc.status == Status.FAILED_PRECONDITION:
-                        raise InvalidError(exc.message)
-                    if exc.message and "Received :status = '413'" in exc.message:
+                except Exception as exc:
+                    if "Received :status = '413'" in str(exc):
```

Comment:
It'll be nice not to read the message from this exception, but I think this error message comes from `nginx`, so we do not have as much flexibility to adjust the message on the server.